feature/google/src/test/java/BillingWrapperTest.kt

```diff
@@ -479,61 +440,15 @@ class BillingWrapperTest {
         }
     }
 
-    @Test
```

Comment:
any tests that are completely deleted from this file should be moved into both of the flavor-specific files now, with their unique implementations